### ISSUE TITLE
Portal Gun locking

### ISSUE THREAD

**Issue by phantomeye1931** [snowy-shack/PortalMod#22]
Locking Portal Guns to a single portal color

**Comment by phantomeye1931** [comment]
This is in survival only useful if auto portals are implemented, as the portals you can place with your locked gun can never open otherwise. 

In creative or using commands it might be nice to have anyway as the other portal can be opened with commands, but this is very niche. Locked guns shouldn't be craftable before auto portals are a thing, or it would cause even more confusion

**Comment by LarsMans64** [comment]
Yeah i agree, it would be nice if locked portals are still obtainable in 1.0 by using creative / commands

**Comment by phantomeye1931** [comment]
Yeah that would definitely be nice
